src/hooks/screen/useSignedChatInfoHook.ts

```diff
@@ -2,37 +2,38 @@ import * as React from 'react';
 import {useNavigation} from '@react-navigation/core';
 
 import ChannelList from '../../database/schema/ChannelListSchema';
-import UseAnonymousChatInfoScreenHook from '../../../types/hooks/screens/useAnonymousChatInfoScreenHook.types';
+import UseSignedChatInfoScreenHook from '../../../types/hooks/screens/useSignedChatInfoScreenHook.types';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import {ChannelListMemberSchema} from '../../../types/database/schema/ChannelList.types';
 import {getAnonymousUserId} from '../../utils/users';
 import {getUserId} from '../../utils/token';
 
-function useAnonymousChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
+function useSignedChatInfoScreenHook(): UseSignedChatInfoScreenHook {
   const {localDb, channelList} = useLocalDatabaseHook();
   const {selectedChannel, goBack} = useChatUtilsHook();
-
   const navigation = useNavigation();
 
   const [channelInfo, setChannelInfo] = React.useState(null);
 
-  const initChatInfoData = async () => {
+  const initializeChatInfoData = async () => {
     if (!localDb) return;
     const myId = await getUserId();
+
     const myAnonymousId = await getAnonymousUserId();
     const data = await ChannelList.getChannelInfo(
       localDb,
       selectedChannel?.id,
       myId,
       myAnonymousId
     );
+
     setChannelInfo(data);
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `initializeChatInfoData` is a new addition. It seems to be replacing the old `initChatInfoData` function. The logic appears to be the same, but ensure that the new function name is used consistently throughout the codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->